ports/psoc6/modules/machine/pins.h

```diff
@@ -0,0 +1,21 @@
+// std includes
+#include <stdio.h>
+#include <stdlib.h>
+
+// port-specific includes
+#include "modmachine.h"
+
+
+/* Include Pin definitions to make it visible at py side*/
+#define PIN_P13_7 CYHAL_GET_GPIO(CYHAL_PORT_13, 7)
```

Comment:
Why these 2 pins and not all ports ?

Reply:
It was planned that these files `pins.c` and `pins.h` will be auto-generated after consuming board-specific structures (csv files, enums etc). We do not have that infrastructure in place yet. Hence this was manually written for the current board. It can be extended manually, if need be, for the current board.

Reply:
If we plan to put this version on github, then the list of pins should be extended. Otherwise we should better not put it on github. Doing it manually, i.e. text replacement, is probably quicker than writing a script for this demonstrator release..

Reply:
Added all the pins manually for the package on the CYPROTO 062-4343W board. To be auto-generated later since certain aspects can vary from board to board (package to package).